test/functional/fixtures/api/es-next/roles/test.js

```diff
@@ -112,4 +112,39 @@ const isRemoteTask = config.currentEnvironmentName === config.testingEnvironment
                 });
         });
     });
+
+    describe('URL is Role constructor', function () {
+        it('Should throw "error in role initializer" without baseUrl and with relative path Role', () => {
+            return runTests(
+                './testcafe-fixtures/role-with-baseurl-test.js',
+                'Should throw error in role initializer without baseUrl and with relative path Role',
+                {
+                    shouldFail: true,
+                }
+            )
```

Comment:
Please align it. Closed brace and `.catch` should be in one column. I suggest moving rows 119-124 to the right.